### PR TITLE
Required updates for Cropper.

### PR DESCRIPTION
**Pull request by deptant**
This update includes templates for media-library. There are css updates to reflect the modal changes in both the backend and the front end used by layout builder.

Early iterations of the theme used some form templates which was bad practise and caused many issues with media library and cropper. These have been renamed to -old. I have left them in place, in case we notice any issues caused by removing them. If this happens, they should be renamed/replaced with more specific templates using template suggestions.

Specific templates have been introduced for the cropper image in Layout Builder as the front end was causing issues.

### CODE REVIEW

File: css/ckeditor-override.css [hunk 1]
```diff
@@ -82,3 +82,147 @@ ol ol {
 .uds-tooltip-dark .fa-stack .fa-info {
   color: #191919;
 }
+<<<<<<< Updated upstream
+=======
```
Comment (mlsamuelson) [inline css/ckeditor-override.css:86]: @duarte-daniela Looks like some Git conflict for some other sort of Git formatting here.

File: css/ckeditor-override.css [hunk 2]
```diff
@@ -82,3 +82,147 @@ ol ol {
...
+
+  .responsive-enabled--opened .vertical-tabs__menu {
+    width: 206px;
+  }
+
+  .responsive-enabled--opened .vertical-tabs__panes {
+    margin-left: 205px;
+  }
+}
+>>>>>>> Stashed changes
```
Comment (mlsamuelson) [inline css/ckeditor-override.css:228]: @duarte-daniela Another bit that needs cleanup.

**Review by mlsamuelson — CHANGES_REQUESTED** [review]
@duarte-daniela Looks like there is some Git metadata formatting that snuck into css/ckeditor-override.css. See the comments I've added in the content.

**Review by ovista-dept — APPROVED** [review]
I removed the unwanted get formatting charters. This looks good to me.

**Review by mlsamuelson — APPROVED** [review]
Thanks @ovista-dept !

**Pull request merged**